### ISSUE TITLE
Touch ID 

### ISSUE THREAD

**Issue by thedannymullen** [chadaustin/passdrop#6]
Any chance we could add Touch ID or pin to auto open database?

You would enter the master key once then it could use Touch ID or pin in future.

Minikeepass has it I don’t know if that source code routines can be imported or not.

**Comment by chadaustin** [comment]
I agree that would be a great feature. The next time I have a chance to go through the process of pushing a new release into the app store, I'll look into what it would take.

**Comment by thedannymullen** [comment]
@chadaustin Thanks! That would be amazing!  Great product you wrote!

**Comment by chadaustin** [comment]
Thanks! I can't take credit for creating it though. I just took over maintainership from @rudism. :)

**Comment by thedannymullen** [comment]
Yes, but maintaining deserves credit also!!!  So thanks!

**Comment by GatorBate** [comment]
Chad Austin, I know LIFE is CrAZy BusY!!!!  Your TIME IS VALUABLE!!!  If you CAN LEGALLY set-up a LIFETIME APP - Purchase for a reasonable fee to pay you for the time you would spend, basically past & future WOULD YOU PLEASE CONSIDER ADDING this feature & giving more of your time supporting & maintaining this APP???   Especially being they have REMOVED MINIKEEPASS as of the end of this year 2020.  I TRULY BELIEVE, THOUSANDS of people will be seeking THE CLOSET APP to REPLACE that APP for a Reasonable Fee!!!  You WILL HAVE LIFETIME SUPPORTERS!!!  Lifetime!!  I WOULD BELIEVE A CHARGE OF $50 or less would be the most YOU COULD CHARGE!  Strongbox has a FAMILY APP FOR $54 LIFETIME which is VERY CLOSE TO KEEPASS & THIS APP, EXCEPT it has a simple way to convert it to CSV to Print for SAFE KEEPING in a VAULT or SAFE!                                                                                    PLEASE Consider this As Soon As Possible would be ideal but surely by the Fall I would say.  I currently am NOT allowed to use your APP!!  I would need your support!!!  I am typing the exact correct Master Password but it is acting as though I am not!  I use the same password for all the other APP’s I have tried & it DID WORK FINE, so I know its correct.  If you know of an issue with IOS 13.4.4 & KBDX 4 or KBDX 3 files....let me know!  I will check this site!                       GOD 🕊Bless you & yours always 🙏🏼🕊 DURING THESE TIMES!

**Comment by chadaustin** [comment]
Closing all issues because this app is no longer supported.
